pages/1.13/administering-clusters/deploying-a-local-dcos-universe/index.md

```diff
@@ -249,9 +250,9 @@ Repeat this section until you have completed this procedure for all of your mast
 
 **Prerequisite:** [Git](https://git-scm.com/). On Unix/Linux, see these <a href="https://git-scm.com/book/en/v2/Getting-Started-Installing-Git" target="_blank">installation instructions</a>.
 
-To deploy a local Universe containing your own set of packages you must build a customized local Universe Docker image.
+To deploy a local {{ model.packageRepo }} containing your own set of packages you must build a customized local {{ model.packageRepo }} Docker image.
```

Comment:
This should be referred to as `Local Universe` all through out this file.

Reply:
See Slack convo.